<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RgbDialog</class>
 <widget class="QDialog" name="RgbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>211</width>
     <height>252</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="redLabel">
        <property name="text">
         <string>Red:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="redInput">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="redSlider">
      <property name="sliderPosition">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="greenLabel">
        <property name="text">
         <string>Green:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="greenInput"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="greenSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="blueLabel">
        <property name="text">
         <string>Blue:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="blueInput"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="blueSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="confRgbBtn">
        <property name="text">
         <string>confirm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>RgbDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confRgbBtn</sender>
   <signal>clicked()</signal>
   <receiver>RgbDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
